core/templates/dev/head/galleries/Gallery.js

```diff
@@ -203,6 +203,51 @@ oppia.controller('Gallery', [
       $rootScope.loadingMessage = '';
     });
 
+    $scope.leftCheck = function(ind) {
+      var tilesHtml = '.oppia-gallery-tiles:eq(n)'.replace('n', ind);
+
+      // Checks if it is at the left most point.
+      return parseInt($(tilesHtml).css('left')) >= 0;
+    };
+
+    $scope.scrollLeft = function(ind) {
+      if (!$scope.leftCheck(ind)) {
+        var tilesHtml = '.oppia-gallery-tiles:eq(n)'.replace('n', ind);
```

Comment:
Might want to make this into a separate function (declared as a var, not on $scope): getJQuerySelectorForGalleryGroup(ind)